themes/src/main/resources/theme/keycloak.v2/account/src/tsconfig.json

```
@@ -8,10 +8,11 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "lib": [ "es2015", "dom", "dom.iterable" ],
-    "noImplicitAny": true,
-    "strictNullChecks": true,
+    "noImplicitAny": false,
+    "strictNullChecks": false,
```

We should keep the TypeScript configuration as strict as possible.

```suggestion
    "noImplicitAny": true,
    "strictNullChecks": true,
```